src/views/PageUserProfile.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div>
+        <TemplateUserProfile></TemplateUserProfile>
```

Comment:
creating a template with just one component.
should we have broken it differently?

Reply:
this page is removed